app/services/provider_interface/hesa_data_export.rb

```diff
@@ -1,22 +1,26 @@
 module ProviderInterface
   class HesaDataExport
-    def initialize(provider_ids:)
-      @provider_ids = provider_ids
+    def initialize(actor:)
+      @provider_ids = actor.providers.map(&:id)
+      @auth = ProviderAuthorisation.new(actor: actor)
```

Comment:
I think `@actor.authorisation` will do the job here as this will always be a `ProviderUser`?